specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -3100,6 +3100,9 @@
           "Create a Linux vm with a patch setting patchMode of AutomaticByPlatform.": {
             "$ref": "./examples/CreateALinuxVmWithPatchSettingModeOfAutomaticByPlatform.json"
           },
```

Comment:
The title of this PR indicates these changes should be in the 2021-03-01 version. If so, please make this PR against the 2021-03-01 compute.json file and move the examples to the 2021-03-01 Examples folder.

Reply:
oh wow what a mistake how did i miss that